src/lib/three.js

```diff
@@ -1,11 +1,27 @@
-var THREE = require('./three.mjs').default;
+import * as SUPER_THREE from 'three';
+import { DRACOLoader } from 'three/examples/jsm/loaders/DRACOLoader.js';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+import { KTX2Loader } from 'three/examples/jsm/loaders/KTX2Loader.js';
+import { OBB } from 'three/addons/math/OBB.js';
+import { OBJLoader } from 'three/examples/jsm/loaders/OBJLoader.js';
+import { MTLLoader } from 'three/examples/jsm/loaders/MTLLoader.js';
+import * as BufferGeometryUtils from 'three/examples/jsm/utils/BufferGeometryUtils.js';
+import { LightProbeGenerator } from 'three/examples/jsm/lights/LightProbeGenerator.js';
+import { DeviceOrientationControls } from '../../vendor/DeviceOrientationControls.js'; // THREE.DeviceOrientationControls
+
+var THREE = global.THREE = SUPER_THREE;
+
 // TODO: Eventually include these only if they are needed by a component.
-global.THREE = THREE;
-require('../../vendor/DeviceOrientationControls');
+THREE.DRACOLoader = DRACOLoader;
```

Comment:
Before we had
```js
var THREE = { ...SUPER_THREE };
```
that was needed for node that you can test with `npm run test:node`
you get `TypeError: Cannot add property DRACOLoader, object is not extensible`
without it.

Reply:
We also have the error with webpack `output.libraryTarget: 'module'` so we need it back.
Also we should use window instead of global for a future no build context. global works only via webpack or node.

```js
var THREE = window.THREE = {...SUPER_THREE};
```

Reply:
Now I recall having it that way before, but changing it while testing tree-shaking, as `{...SUPER_THREE}` prevents most tree-shaking. But indeed, it should be `{...SUPER_THREE}` for the time being.

Ultimately I think it would be better to have the A-Frame code import from `three` and its addons directly where applicable. Then construct this "SUPER_THREE + some addons" only for the final exports and populating the global scope. But that is beyond the scope of this PR.

Reply:
Yes I totally agree. For example importing `LightProbeGenerator` should really be done in `src/components/light.js`. We can do that in a separate PR.